server/test/suite/fireboltOpenRpc.test.mjs

```diff
@@ -0,0 +1,735 @@
+      name: "rpc.discover",
+      summary: "Firebolt OpenRPC schema",
+      params: [],
+      result: {
+        name: "OpenRPC Schema",
+        schema: {
+          type: "object",
+        },
+      },
+    },
+    "accessibility.voiceGuidanceSettings": {
+      name: "accessibility.voiceGuidanceSettings",
+      summary: "Get the user's preferred voice guidance settings",
+      params: [],
+      tags: [
+        {
+          name: "property:readonly",
+        },
+      ],
+      result: {
+        name: "settings",
+        summary: "the voice guidance settings",
+        schema: {
+          $ref: "#/components/schemas/VoiceGuidanceSettings",
+        },
+      },
+      examples: [
+        {
+          name: "Getting the voice guidance settings",
+          params: [],
+          result: {
+            name: "Default Result",
+            value: {
+              enabled: true,
+              speed: 5,
+            },
+          },
+        },
+      ],
+    },
+  };
+  const expectedInput = [
+    "rpc.discover",
+    "accessibility.voiceGuidanceSettings",
+    "invalidMethod",
+  ];
+  const out = fireboltOpenRpc.testExports.methodMaps.core;
+  const expectedOutput = [
+    out["rpc.discover"],
+    out["accessibility.voiceGuidanceSettings"],
+    undefined,
+  ];
+  expectedInput.forEach((methodName, index) => {
+    const result = fireboltOpenRpc.getMethod(methodName);
+    expect(result).toEqual(expectedOutput[index]);
+  });
+});
+
+test(`fireboltOpenRpc.isMethodKnown works properly`, () => {
+  fireboltOpenRpc.testExports.methodMaps["core"] = {
+    "rpc.discover": {
+      name: "rpc.discover",
+      summary: "Firebolt OpenRPC schema",
+      params: [],
+      result: {
+        name: "OpenRPC Schema",
+        schema: {
+          type: "object",
+        },
+      },
+    },
+    "accessibility.voiceGuidanceSettings": {
+      name: "accessibility.voiceGuidanceSettings",
+      summary: "Get the user's preferred voice guidance settings",
+      params: [],
+      tags: [
+        {
+          name: "property:readonly",
+        },
+      ],
+      result: {
+        name: "settings",
+        summary: "the voice guidance settings",
+        schema: {
+          $ref: "#/components/schemas/VoiceGuidanceSettings",
+        },
+      },
+      examples: [
+        {
+          name: "Getting the voice guidance settings",
+          params: [],
+          result: {
+            name: "Default Result",
+            value: {
+              enabled: true,
+              speed: 5,
+            },
+          },
+        },
+      ],
+    },
+  };
+  const expectedInput = [
+    "rpc.discover",
+    "accessibility.voiceGuidanceSettings",
+    "invalidMethod",
+  ];
+  const expectedOutput = [true, true, false];
+  expectedInput.forEach((methodName, index) => {
+    const result = fireboltOpenRpc.isMethodKnown(methodName);
+    expect(result).toBe(expectedOutput[index]);
+  });
+});
+
+// test('fireboltOpenRpc.getSchema', () => {
```

Comment:
![getSchema_error](https://user-images.githubusercontent.com/102782078/179217195-4af219f1-3fe4-4200-860f-3c3bf420aa79.png)

@michaelcfine, getting this error while running the test case.